app/lib/bk/compat/parsers/circleci.rb

```diff
@@ -65,6 +62,10 @@ def parse
 
       private
 
+      def register_translators
+        BK::Compat::CircleCISteps::Builtins.new(register: method(:register_translator), recursor: method(:translate_steps))
```

Comment:
same here:
```suggestion
        BK::Compat::CircleCISteps::Builtins.new(
          register: method(:register_translator),
          recursor: method(:translate_steps)
        )
```

Reply:
Have reverted to the multiline way (agreed that its easier on the eyes!)